.github/workflows/build.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Build
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches:
+      - master
+jobs:
+  container:
+    name: Build container
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v4
+      - name: Login to dockerhub
+        uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_READONLY_TOKEN }}
```

Comment:
why readonly ?

Reply:
then maybe we dont need to login to the docker?   I am trying to keep the minikubebot hits under 200 per 6 hours

Reply:
Without login we may fail with rate limit. If we run this only for PRs, this not be a problem.